tests/testsuite/lints.rs

```diff
@@ -0,0 +1,177 @@
+use cargotest::support::{execs, project};
```

Comment:
The `cargotest::support` module is now just `support`, as of #5762.

Reply:
Yeah it does because CI tests the merge of this PR branch with the (current) state of the target branch (where the merge has happened).